src/poli_sci_kit/utils.py

```diff
@@ -141,17 +141,20 @@ def arc_coordinates(r, seats):
             Generates an arc of the parliament plot given a radius and the number of seats.
             """
             angles = np.linspace(start=np.pi, stop=0, num=seats)
-            x_coordinates, y_coordinates = [], []
 
             # Broadcast angles to their corresponding coordinates.
             x_coordinates = list(r * np.cos(angles))
             y_coordinates = list(r * np.sin(angles))
 
-            return x_coordinates, y_coordinates
+            return x_coordinates, y_coordinates, list(angles)
 
-        xs, ys = [], []
+        # Store point coordinates (x, y) and their angles with origin (0, 0)
```

Comment:
Minor nitpick, with Python it's customary that full line comments are complete sentences with punctuation :) For in-line comments there's no punctuation and the first word should not be capitalized 😊 Just FYI, I can get these!